charts/sonarqube-dce/Chart.yaml

```diff
@@ -19,32 +19,9 @@ maintainers:
 annotations:
   artifacthub.io/changes: |
     - kind: changed
-      description: "Helm charts repository moved to a new location"
-      links:
-        - name: New location
-          url: TBD
-    - kind: changed
-      description: "changed external database connection logic"
-    - kind: added
-      description: "added option to use external secret as jwtSecret"
-    - kind: fixed
-      description: "removed initSysctl from application pods"
+      description: "Homoginized Description of the Helm Chart"
```

Comment:
I don't understand. This is a change log section?

Reply:
Yep this annotation is used on artifacthub to render the changelog: https://artifacthub.io/packages/helm/sonarqube/sonarqube?modal=changelog